priv/perf/apps/load_test/lib/ethereum/bit_helper.ex

```diff
@@ -27,7 +27,7 @@ defmodule LoadTest.Ethereum.BitHelper do
   """
   @spec kec(binary()) :: keccak_hash
   def kec(data) do
-    :keccakf1600.sha3_256(data)
+    elem(ExKeccak.hash_256(data), 1)
```

Comment:
what do you think about raising the error?

Reply:
`perf` should move to the latest ex_plasma too so lets address it when it comes to that